solar/tests/test_solar.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import os.path
+sys.path.append(os.path.dirname(sys.path[0]))
+
+from solar import SolarDay      
+from cell import SolarCell
+
+global solar 
+solar = SolarDay(1,2,3,4,5,6) 
```

Comment:
Could we rename this something more specific? especially since solar is name of package